Makefile

```diff
@@ -1,95 +1,160 @@
+targets = \
+  help \
+  \
+  test \
+  coverage \
+  coverage-html \
+  code-check \
+  \
+  update-schema \
+  data-migration \
+  migrations \
+  migrate \
+  models \
+  \
+  status \
+  checkout \
+  \
+  run-server \
+  stop-server \
+  shutdown-vms \
+  delete-vms \
+  build \
+  run-all-servers \
+  stop-all-servers \
+  shutdown-all-vms \
+  delete-all-vms \
+  build-all \
+  \
+  bash-shell \
+  db-shell \
+  django-shell \
+  \
+  load-db \
```

Comment:
thanks for adding back `load-remote-db`. If possible, it's better to add it to this list, as well :)